wikidatarefisland/external_identifiers/generate_whitelisted_ext_ids.py

```diff
@@ -0,0 +1,85 @@
+import concurrent.futures
+
+import requests
+
+
+class GenerateWhitelistedExtIds():
+    def __init__(self, wdqs_reader, storage, config, no_checks=10, no_processes=10):
+        """
+        Implementation of Side-step 1
+
+        :type wdqs_reader: wikidatarefisland.data_access.WdqsReader
+        :type storage: wikidatarefisland.data_access.Storage
+        :type config: wikidatarefisland.Config
+        """
+        self.wdqs_reader = wdqs_reader
+        self.storage = storage
+        self.config = config
+        self.no_checks = no_checks
+        self.no_processes = no_processes
+        self.result_file_name = 'ext_ids_check_result.json'
+
+    def check_cases(self, usecases, formatter_urls_for_id):
+        total_number = 0
+        good_responses = 0
+        schema_org_responses = 0
+        for case in usecases:
+            total_number += 1
+            value = case['value']['value']
+            url = formatter_urls_for_id[0].replace('$1', value)
+            try:
+                r = requests.get(url, timeout=30)
+            except KeyboardInterrupt:
+                raise
+            except:  # noqa: E722
+                continue
+            if r.status_code == 200:
+                good_responses += 1
+
+            # TODO: Proper check
+            if 'http://schema.org' in r.text:
+                schema_org_responses += 1
+        return total_number, good_responses, schema_org_responses
```

Comment:
Circling back to the question above about triplets, do we really need to store all this data?

Reply:
I turned it to a dictionary. I think it would be useful to know if the response just doesn't have schema.org data or it's just fully down.